service/src/test/java/bio/terra/workspace/pact/TpsApiTest.java

```diff
@@ -35,6 +35,8 @@
 @Tag("pact-test")
 @ExtendWith(PactConsumerTestExt.class)
 public class TpsApiTest {
+  private static final String UUID_REGEX =
```

Comment:
The `UUID_REGEX` was previously provided by the `dsl.DslPart` static import.  However, the new version of pact seems to have rewritten a bunch of the Kotlin innards, and the `UUID_REGEX` is no hidden in a companion object which hasn't been made visible to the Java runtime.